### PR TITLE
feature/OC-197/breadcrumbs

### PR DESCRIPTION
**Pull request by RalkeyOfficial**
Added breadcrumbs

### CODE REVIEW

File: pwa/src/templates/breadcrumb/BreadcrumbTemplate.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  const uppercaseCrumbs = crumbs.map((crumb: any) => ({
+    ...crumb,
+    crumbLabel: _.upperFirst(crumb.crumbLabel),
+  }));
+
+  const translatedBreadcrumbs = uppercaseCrumbs.map((crumb: any) => ({
+    ...crumb,
+    crumbLabel: t(crumb.crumbLabel),
+  }));
+
```
Comment (remko48) [inline pwa/src/templates/breadcrumb/BreadcrumbTemplate.tsx:26]: Nice job of translating the breadcrumbs! 
There is an easier way of doing this:
```
  const translatedBreadcrumbs= crumbs.map((crumb: any) => ({
    ...crumb,
    crumbLabel: t(_.upperFirst(crumb.crumbLabel)),
  }));
```
This way you can have both functions in one.

File: pwa/src/pages/documentation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { navigate } from "gatsby";
+import * as React from "react";
+
+const IndexPage = () => {
+  navigate("/");
+  return <></>;
+};
+
+export default IndexPage;
```
Comment (remko48) [inline pwa/src/pages/documentation/index.tsx:9]: Nice catch!
I did notice that it still breaks on the organization's page breadcrumb. I suggest doing the same redirect there as you did here.

**Review by remko48 — CHANGES_REQUESTED** [review]
Looks really good!

Left some small comments.

**Review by lencodes — CHANGES_REQUESTED** [review]
I think it would be nice to have it look like this.

The background could be `--web-app-color-background-light`. Probably needs updates in component package.

image


Final note: I see that Documentatie > Over doesn't show "Documentatie" in the crumbs.

**Review by lencodes — APPROVED** [review]
Nice!

**Pull request merged**